bitcrowd/bitstyles
# [#712] Update typography

Fixes #712 

## Changes

-  Updates the font-sizes and line-heights of the default configuration
- Adds a stories.js file, stories are then imported into the mdx
- Adds a helper function to convert font-sizes from rem to px, so the storybook font-size DocBlock can parse them

## 📸

<img width="1100" alt="Screenshot 2023-04-03 at 16 56 16" src="https://user-images.githubusercontent.com/2479422/229549687-3df6269a-66ed-46e4-a579-bde4eeec453f.png">

## How to QA Visual changes

- `git checkout main` 
- `git pull` 
- `git checkout feature/712-update-typography` 
- `yarn`
- `yarn storybook`
- navigate to design tokens/typography

Check:

- [x] The appearance matches the zeplin designs

## How to QA code changes


## Preflight checks

PR author to check, and delete if not applicable:
